test/functional/api/v2/configs_controller_test.rb

```diff
@@ -158,6 +159,7 @@ class ForemanVirtWhoConfigure::Api::V2::ConfigsControllerTest < ActionController
     assert_equal 'password', new_config.hypervisor_password
     assert_equal true, new_config.debug
     assert_equal 'foreman.example.com', new_config.satellite_url
+    assert_equal @http_proxy.id, ForemanVirtWhoConfigure::Config.find(response['id']).http_proxy_id
```

Comment:
Doesn't the response have the proxy data inside, to avoid having to go and look it up in the db again?

I think it's like response['http_proxy']['id'] based on your change.

Might as well go ahead and add an assertion for the 'proxy' string as well

Reply:
@jturel updated the tests to assert the response as you suggested, they are all passing:

```bash
# Running:

..............................................

Finished in 11.466540s, 4.0117 runs/s, 15.8723 assertions/s.

46 runs, 182 assertions, 0 failures, 0 errors, 0 skips
```